### PR TITLE
✨(frontend) teacher dashboard routes and breadcrumb

### PR DESCRIPTION
**Pull request by rlecellier**
## Purpose

We want the perfect breadcrumb

## Proposal

* clear unused route
* fix the router to use our breadcrumb as it should be.

image
image
image

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -29,6 +29,8 @@ course's listing page.
 - Use css colors from cunningham instead of palette.scss
 - Improve the design of the dashboards layouts.
 - Improve the design of the dashboards sidebars.
+- Fix teacher dashbaord breadcrumb.
+- Remove unused route in teacher dashboard.
```
Comment (jbpenrath) [inline CHANGELOG.md:33]:
```suggestion

### Removed

- Remove unused route in teacher dashboard.
```

Comment (jbpenrath) [inline CHANGELOG.md:33]: Is it possible to split this commit in two ? One about the refactoring of the breadcrumb and a second in charge to remove all stuff related to unused routes ?

Comment (rlecellier) [reply CHANGELOG.md:33]: I had two commit that i squash because it was way easier.
I'll resplit them.

**Review by jbpenrath — APPROVED** [review]
Do not forgot to update the CHANGELOG according to my comment https://github.com/openfun/richie/pull/2003#discussion_r1267026506

Otherwise LGTM ✅

**Pull request merged**